mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -85,6 +85,18 @@ class LinearRegression(override val uid: String)
   setDefault(fitIntercept -> true)
 
   /**
+   * Whether to standardize the training features before fitting the model.
+   * The coefficients of models will be always returned on the original scale,
+   * so it will be transparent for users. Note that when no regularization,
+   * with or without standardization, the models should be always converged to
+   * the same solution.
+   * Default is true.
+   * @group setParam
+   */
+  def setStandardization(value: Boolean): this.type = set(standardization, value)
+  setDefault(standardization -> true)
```

Comment:
Yah I think it would be nice to just mention that while this is a new flag, the old default effectively had this flag on so don't worry (also maybe add a @since tag?)